src/com/dalolorn/sr2modmanager/view/MainController.java

```diff
@@ -38,6 +42,15 @@ public void initialize() {
 			msg.showAndWait();
 			setSR2Path((Window) null);
 		}
+
+		try {
+			Recommendation.load();
+			recommendationList.getItems().addAll(Recommendation.getInstance().getRecommendationList());
+		} catch (IOException e) {
+			Alert msg = new Alert(Alert.AlertType.ERROR, "Could not load history.json! Cause:"  + e.toString());
```

Comment:
This should maybe only be a warning. It won't seriously interfere with the operation of the application.